File: 151004/Tishalovich/rest_basics/src/main/java/by/bsuir/dc/rest_basics/services/impl/mappers/LabelMapper.java
package by.bsuir.dc.rest_basics.services.impl.mappers;

import by.bsuir.dc.rest_basics.entities.Label;
import by.bsuir.dc.rest_basics.entities.dtos.request.LabelRequestTo;
import by.bsuir.dc.rest_basics.entities.dtos.response.LabelResponseTo;
import org.mapstruct.Mapper;

@Mapper
public interface LabelMapper
        extends EntityMapper<LabelRequestTo, LabelResponseTo, Label> {
}

File: 151004/Bordilovsky/DC_JPA/Services/Mappers/LabelMapper.cs
using AutoMapper;
using DC_REST.DTOs.Request;
using DC_REST.DTOs.Response;
using DC_REST.Entities;

namespace DC_REST.Services.Mappers
{
	public class LabelMapper : Profile
	{
		public LabelMapper() 
		{
			CreateMap<Label, LabelResponseTo>();
			CreateMap<LabelRequestTo, Label>();
		}
	}
}
